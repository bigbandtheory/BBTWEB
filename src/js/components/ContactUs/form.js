import React,{Component} from 'react';
import PropTypes from 'prop-types';
import TextField from '../Fields/TextField';

export default class Form extends Component{

  static propTypes ={
    onChangeHandler : PropTypes.func.isRequired
  }


  render(){

    return(
     <form>
      <div className="col-lg-8 col-centered">
        <div className="col-lg-6">
            <TextField name="name" label="Name" required={true} onChangeHandler = {this.props.onChangeHandler}/>
        </div>
        <div className="col-lg-6">
            <TextField name="email" type="email" label="Email" required={true} onChangeHandler = {this.props.onChangeHandler}/>
        </div>
      </div>
      <div className="col-lg-10 col-centered">
        <div className="col-lg-12">
           <TextField name="message" label="Message" required={true} onChangeHandler = {this.props.onChangeHandler}/>
        </div>
       </div>
       {this.props.children}
     </form>
    )
  }


}