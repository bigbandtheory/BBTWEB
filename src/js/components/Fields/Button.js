import React,{Component,PropTypes} from 'react';
import classNames from 'classnames';

export default class Button extends Component{

  static propTypes = {
    onClick : PropTypes.func.isRequired,
    label : PropTypes.string,
    wrapperClass : PropTypes.string,
    btnClass : PropTypes.string,
    url : PropTypes.string,
  }

  static defaultProps = {
    wrapperClass : 'col-lg-12 t-center',
    btnClass : '',
    label : 'SEND MESSAGE'
  }

  constructor(props){
    super(props);
    this.state = {
      disabled : false
    };
  }

  componentDidMount() {
      let forms = document.querySelectorAll('form');

      for (let form of forms) {

        if (form.contains(this.buttonEl)) {
          this.formEl = form;
          break;
        }
      }

      if (this.formEl) {
        this.toggleDisabled();
        this.formEl.addEventListener( 'click', this.toggleDisabled.bind(this));
        this.formEl.addEventListener('input', this.toggleDisabled.bind(this));
      }
    }

    toggleDisabled() {
      setTimeout(() => {
        this.setState({
          disabled: !this.formEl.checkValidity(),
        });
      }, 100);

    }


  render(){
    let btnCls = `${this.props.btnClass} gradientButton pink inverse mTop50`;
    let btnClass =  classNames({
      [btnCls] : true,
      'disable-link' : this.state.disabled
    });

    return(
      <div ref={(x) => this.buttonEl = x} className={this.props.wrapperClass} >
          <a href={this.props.url ? this.props.url : null} onClick ={this.props.onClick} className={btnClass}>{this.props.label}</a>
      </div>
    );
  }

}