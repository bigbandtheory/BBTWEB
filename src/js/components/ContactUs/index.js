import React,{Component, PropTypes} from 'react';
import Form from './form';
import {sendSubscriptionMessage} from '../../network';
import Button from '../Fields/Button';

export default class ContactUs extends Component{

  constructor(props){
    super(props);
    this.state = {
      contactData : {}
    }
  }

  onChangeHandler(event){
    let value = event.target.value;
    this.state.contactData[event.target.name] = value;
  }

  submitMessage(event){
    event.preventDefault();
    sendSubscriptionMessage(this.state.contactData);
  }


   render(){
     return(
        <div className="row pattern-black contact logo-container" id="contact">
          <div className="container">
                <h2 className="sectionTitle f-damion c-white mTop100">
                    Get in Touch!
                </h2>
                <h6 className="f-robotoSlab fw-400 c-turquoise t-center mBottom30">Be the first to know about our Events, Artist, Music</h6>
                <Form onChangeHandler={this.onChangeHandler.bind(this)}>
                    <Button onClick = {this.submitMessage.bind(this)} className="gradientButton pink inverse mTop50" label = "SEND MESSAGE"/>
                </Form>
          </div>
          <div className="container mTop50 mBottom50">
                <div className="col-lg-8 col-sm-10 col-centered">
                    <div className="col-lg-12 col-sm-12">
                        <div className="row">
                            <div className="col-lg-4 col-sm-4 col-xs-12 t-center mBottom40">
                                <i className="flaticon-laptop15 c-pink iconSize45 mBottom20 hvr-buzz-out"></i>
                                <h6 className="f-raleway fw-700 c-white">E-Mail</h6>
                                <p className="c-lightGrey t-center"><a className="textButtonWhite" href="mailto:info@thebigbandtheory.net">info@thebigbandtheory.net</a></p>
                            </div>
                            <div className="col-lg-4 col-sm-4 col-xs-12 t-center mBottom40">
                                <i className="flaticon-smart1 c-pink iconSize45 mBottom20 hvr-buzz-out"></i>
                                <h6 className="f-raleway fw-700 c-white">Phone</h6>
                                <p className="c-lightGrey t-center">+91-8882003835</p>
                            </div>
                            <div className="col-lg-4 col-sm-4 col-xs-12 t-center mBottom40">
                                <i className="flaticon-open79 c-pink iconSize45 mBottom20 hvr-buzz-out"></i>
                                <h6 className="f-raleway fw-700 c-white">Address</h6>
                                <p className="c-lightGrey t-center">190 model colony araghar, <br/>Dehradun, Uttrakhand 248001</p>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
        </div>
     );
   }
}