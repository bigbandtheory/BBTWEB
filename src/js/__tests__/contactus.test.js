import React from 'react';
import ContactUs from '../components/ContactUs';
import Form from '../components/ContactUs/Form';
import {mount, shallow} from 'enzyme';
import sinon from 'sinon';
import renderer from 'react-test-renderer';
import Button from '../components/Fields/Button';


describe('Component Contact Us ',()=>{
  let component = renderer.create(<ContactUs  />);
  test('It should mount', ()=>{
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
    //tree.props.onMouseEnter();
  });

  test('It should simulate submit action ', ()=>{
     let contactUs = shallow(<ContactUs />);
     sinon.spy(ContactUs.prototype, 'submitMessage');
     let form = contactUs.find(Form)
     expect(form.length).toBe(1);
     let button = form.dive().find(Button);
     expect(button.length).toBe(1);
     button.setState({disabled : true});
     expect(button).toBeDefined();
     expect(button.length).toBe(1);
     expect(button.props().label).toBe('SEND MESSAGE');

     button.find('a.gradientButton').get(0).simulate('click');
     expect(ContactUs.prototype.submitMessage).toHaveProperty('callCount', 1);
  });

  describe('Contact Us Form',()=>{

    let onChangeHandler = sinon.spy();
    let form = mount(<Form onChangeHandler= {onChangeHandler}></Form>);

    test('It should mount with 3 input type ',()=>{
      expect(Form.prototype.render.calledOnce).toBe.true;
      let input = form.find('input');
      expect(input).toBeDefined();
      expect(input.length).toBe(3);
    });

    test('It simulates on Change functionality',() =>{
      let input = form.find('input[name="name"]');
      expect(input).toBeDefined();
      expect(input.length).toBe(1);
      input.simulate('change');
      expect(onChangeHandler.calledOnce).toBe.true;
    });


    test('On change of element state value should be changed', ()=>{
       let input = form.find('input[name="name"]');
       expect(input).toBeDefined();
       expect(input.length).toBe(1);

    });

  });

});




