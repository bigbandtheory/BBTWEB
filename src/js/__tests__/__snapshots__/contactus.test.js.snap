// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Contact Us  Contact Us should mount 1`] = `
<div
  className="row pattern-black contact logo-container"
  id="contact"
>
  <div
    className="container"
  >
    <h2
      className="sectionTitle f-damion c-white mTop100"
    >
      Get in Touch!
    </h2>
    <h6
      className="f-robotoSlab fw-400 c-turquoise t-center mBottom30"
    >
      Be the first to know about our Events, Artist, Music
    </h6>
    <form>
      <div
        className="col-lg-8 col-centered"
      >
        <div
          className="col-lg-6"
        >
          <span
            className="input input--kozakura"
            style={
              Object {
                "height": "auto",
                "minHeight": 120,
              }
            }
          >
            <input
              className="input__field input__field--kozakura"
              id={undefined}
              maxLength={25000}
              name="name"
              onBlur={[Function]}
              onChange={[Function]}
              onFocus={[Function]}
              onKeyUp={undefined}
              pattern={undefined}
              placeholder=""
              required={true}
              tabIndex={undefined}
              type="text"
            />
            <label
              className="input__label input__label--kozakura"
              htmlFor="input-9"
            >
              <span
                className="input__label-content input__label-content--kozakura"
                data-content="Name"
              >
                Name
              </span>
            </label>
            <svg
              className="graphic graphic--kozakura"
              height="100%"
              preserveAspectRatio="none"
              viewBox="0 0 1200 60"
              width="300%"
            >
              <path
                d="M1200,9c0,0-305.005,0-401.001,0C733,9,675.327,4.969,598,4.969C514.994,4.969,449.336,9,400.333,9C299.666,9,0,9,0,9v43c0,0,299.666,0,400.333,0c49.002,0,114.66,3.484,197.667,3.484c77.327,0,135-3.484,200.999-3.484C894.995,52,1200,52,1200,52V9z"
              />
            </svg>
          </span>
        </div>
        <div
          className="col-lg-6"
        >
          <span
            className="input input--kozakura"
            style={
              Object {
                "height": "auto",
                "minHeight": 120,
              }
            }
          >
            <input
              className="input__field input__field--kozakura"
              id={undefined}
              maxLength={25000}
              name="email"
              onBlur={[Function]}
              onChange={[Function]}
              onFocus={[Function]}
              onKeyUp={undefined}
              pattern={undefined}
              placeholder=""
              required={true}
              tabIndex={undefined}
              type="email"
            />
            <label
              className="input__label input__label--kozakura"
              htmlFor="input-9"
            >
              <span
                className="input__label-content input__label-content--kozakura"
                data-content="Email"
              >
                Email
              </span>
            </label>
            <svg
              className="graphic graphic--kozakura"
              height="100%"
              preserveAspectRatio="none"
              viewBox="0 0 1200 60"
              width="300%"
            >
              <path
                d="M1200,9c0,0-305.005,0-401.001,0C733,9,675.327,4.969,598,4.969C514.994,4.969,449.336,9,400.333,9C299.666,9,0,9,0,9v43c0,0,299.666,0,400.333,0c49.002,0,114.66,3.484,197.667,3.484c77.327,0,135-3.484,200.999-3.484C894.995,52,1200,52,1200,52V9z"
              />
            </svg>
          </span>
        </div>
      </div>
      <div
        className="col-lg-10 col-centered"
      >
        <div
          className="col-lg-12"
        >
          <span
            className="input input--kozakura"
            style={
              Object {
                "height": "auto",
                "minHeight": 120,
              }
            }
          >
            <input
              className="input__field input__field--kozakura"
              id={undefined}
              maxLength={25000}
              name="message"
              onBlur={[Function]}
              onChange={[Function]}
              onFocus={[Function]}
              onKeyUp={undefined}
              pattern={undefined}
              placeholder=""
              required={true}
              tabIndex={undefined}
              type="text"
            />
            <label
              className="input__label input__label--kozakura"
              htmlFor="input-9"
            >
              <span
                className="input__label-content input__label-content--kozakura"
                data-content="Message"
              >
                Message
              </span>
            </label>
            <svg
              className="graphic graphic--kozakura"
              height="100%"
              preserveAspectRatio="none"
              viewBox="0 0 1200 60"
              width="300%"
            >
              <path
                d="M1200,9c0,0-305.005,0-401.001,0C733,9,675.327,4.969,598,4.969C514.994,4.969,449.336,9,400.333,9C299.666,9,0,9,0,9v43c0,0,299.666,0,400.333,0c49.002,0,114.66,3.484,197.667,3.484c77.327,0,135-3.484,200.999-3.484C894.995,52,1200,52,1200,52V9z"
              />
            </svg>
          </span>
        </div>
      </div>
      <div
        className="col-lg-12 t-center"
      >
        <a
          className=" gradientButton pink inverse mTop50"
          href={null}
          onClick={[Function]}
        >
          SEND MESSAGE
        </a>
      </div>
    </form>
  </div>
  <div
    className="container mTop50 mBottom50"
  >
    <div
      className="col-lg-8 col-sm-10 col-centered"
    >
      <div
        className="col-lg-12 col-sm-12"
      >
        <div
          className="row"
        >
          <div
            className="col-lg-4 col-sm-4 col-xs-12 t-center mBottom40"
          >
            <i
              className="flaticon-laptop15 c-pink iconSize45 mBottom20 hvr-buzz-out"
            />
            <h6
              className="f-raleway fw-700 c-white"
            >
              E-Mail
            </h6>
            <p
              className="c-lightGrey t-center"
            >
              <a
                className="textButtonWhite"
                href="mailto:info@thebigbandtheory.net"
              >
                info@thebigbandtheory.net
              </a>
            </p>
          </div>
          <div
            className="col-lg-4 col-sm-4 col-xs-12 t-center mBottom40"
          >
            <i
              className="flaticon-smart1 c-pink iconSize45 mBottom20 hvr-buzz-out"
            />
            <h6
              className="f-raleway fw-700 c-white"
            >
              Phone
            </h6>
            <p
              className="c-lightGrey t-center"
            >
              +91-8882003835
            </p>
          </div>
          <div
            className="col-lg-4 col-sm-4 col-xs-12 t-center mBottom40"
          >
            <i
              className="flaticon-open79 c-pink iconSize45 mBottom20 hvr-buzz-out"
            />
            <h6
              className="f-raleway fw-700 c-white"
            >
              Address
            </h6>
            <p
              className="c-lightGrey t-center"
            >
              190 model colony araghar, 
              <br />
              Dehradun, Uttrakhand 248001
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Component Contact Us  It should mount 1`] = `
<div
  className="row pattern-black contact logo-container"
  id="contact"
>
  <div
    className="container"
  >
    <h2
      className="sectionTitle f-damion c-white mTop100"
    >
      Get in Touch!
    </h2>
    <h6
      className="f-robotoSlab fw-400 c-turquoise t-center mBottom30"
    >
      Be the first to know about our Events, Artist, Music
    </h6>
    <form>
      <div
        className="col-lg-8 col-centered"
      >
        <div
          className="col-lg-6"
        >
          <span
            className="input input--kozakura"
            style={
              Object {
                "height": "auto",
                "minHeight": 120,
              }
            }
          >
            <input
              className="input__field input__field--kozakura"
              id={undefined}
              maxLength={25000}
              name="name"
              onBlur={[Function]}
              onChange={[Function]}
              onFocus={[Function]}
              onKeyUp={undefined}
              pattern={undefined}
              placeholder=""
              required={true}
              tabIndex={undefined}
              type="text"
            />
            <label
              className="input__label input__label--kozakura"
              htmlFor="input-9"
            >
              <span
                className="input__label-content input__label-content--kozakura"
                data-content="Name"
              >
                Name
              </span>
            </label>
            <svg
              className="graphic graphic--kozakura"
              height="100%"
              preserveAspectRatio="none"
              viewBox="0 0 1200 60"
              width="300%"
            >
              <path
                d="M1200,9c0,0-305.005,0-401.001,0C733,9,675.327,4.969,598,4.969C514.994,4.969,449.336,9,400.333,9C299.666,9,0,9,0,9v43c0,0,299.666,0,400.333,0c49.002,0,114.66,3.484,197.667,3.484c77.327,0,135-3.484,200.999-3.484C894.995,52,1200,52,1200,52V9z"
              />
            </svg>
          </span>
        </div>
        <div
          className="col-lg-6"
        >
          <span
            className="input input--kozakura"
            style={
              Object {
                "height": "auto",
                "minHeight": 120,
              }
            }
          >
            <input
              className="input__field input__field--kozakura"
              id={undefined}
              maxLength={25000}
              name="email"
              onBlur={[Function]}
              onChange={[Function]}
              onFocus={[Function]}
              onKeyUp={undefined}
              pattern={undefined}
              placeholder=""
              required={true}
              tabIndex={undefined}
              type="email"
            />
            <label
              className="input__label input__label--kozakura"
              htmlFor="input-9"
            >
              <span
                className="input__label-content input__label-content--kozakura"
                data-content="Email"
              >
                Email
              </span>
            </label>
            <svg
              className="graphic graphic--kozakura"
              height="100%"
              preserveAspectRatio="none"
              viewBox="0 0 1200 60"
              width="300%"
            >
              <path
                d="M1200,9c0,0-305.005,0-401.001,0C733,9,675.327,4.969,598,4.969C514.994,4.969,449.336,9,400.333,9C299.666,9,0,9,0,9v43c0,0,299.666,0,400.333,0c49.002,0,114.66,3.484,197.667,3.484c77.327,0,135-3.484,200.999-3.484C894.995,52,1200,52,1200,52V9z"
              />
            </svg>
          </span>
        </div>
      </div>
      <div
        className="col-lg-10 col-centered"
      >
        <div
          className="col-lg-12"
        >
          <span
            className="input input--kozakura"
            style={
              Object {
                "height": "auto",
                "minHeight": 120,
              }
            }
          >
            <input
              className="input__field input__field--kozakura"
              id={undefined}
              maxLength={25000}
              name="message"
              onBlur={[Function]}
              onChange={[Function]}
              onFocus={[Function]}
              onKeyUp={undefined}
              pattern={undefined}
              placeholder=""
              required={true}
              tabIndex={undefined}
              type="text"
            />
            <label
              className="input__label input__label--kozakura"
              htmlFor="input-9"
            >
              <span
                className="input__label-content input__label-content--kozakura"
                data-content="Message"
              >
                Message
              </span>
            </label>
            <svg
              className="graphic graphic--kozakura"
              height="100%"
              preserveAspectRatio="none"
              viewBox="0 0 1200 60"
              width="300%"
            >
              <path
                d="M1200,9c0,0-305.005,0-401.001,0C733,9,675.327,4.969,598,4.969C514.994,4.969,449.336,9,400.333,9C299.666,9,0,9,0,9v43c0,0,299.666,0,400.333,0c49.002,0,114.66,3.484,197.667,3.484c77.327,0,135-3.484,200.999-3.484C894.995,52,1200,52,1200,52V9z"
              />
            </svg>
          </span>
        </div>
      </div>
      <div
        className="col-lg-12 t-center"
      >
        <a
          className=" gradientButton pink inverse mTop50"
          href={null}
          onClick={[Function]}
        >
          SEND MESSAGE
        </a>
      </div>
    </form>
  </div>
  <div
    className="container mTop50 mBottom50"
  >
    <div
      className="col-lg-8 col-sm-10 col-centered"
    >
      <div
        className="col-lg-12 col-sm-12"
      >
        <div
          className="row"
        >
          <div
            className="col-lg-4 col-sm-4 col-xs-12 t-center mBottom40"
          >
            <i
              className="flaticon-laptop15 c-pink iconSize45 mBottom20 hvr-buzz-out"
            />
            <h6
              className="f-raleway fw-700 c-white"
            >
              E-Mail
            </h6>
            <p
              className="c-lightGrey t-center"
            >
              <a
                className="textButtonWhite"
                href="mailto:info@thebigbandtheory.net"
              >
                info@thebigbandtheory.net
              </a>
            </p>
          </div>
          <div
            className="col-lg-4 col-sm-4 col-xs-12 t-center mBottom40"
          >
            <i
              className="flaticon-smart1 c-pink iconSize45 mBottom20 hvr-buzz-out"
            />
            <h6
              className="f-raleway fw-700 c-white"
            >
              Phone
            </h6>
            <p
              className="c-lightGrey t-center"
            >
              +91-8882003835
            </p>
          </div>
          <div
            className="col-lg-4 col-sm-4 col-xs-12 t-center mBottom40"
          >
            <i
              className="flaticon-open79 c-pink iconSize45 mBottom20 hvr-buzz-out"
            />
            <h6
              className="f-raleway fw-700 c-white"
            >
              Address
            </h6>
            <p
              className="c-lightGrey t-center"
            >
              190 model colony araghar, 
              <br />
              Dehradun, Uttrakhand 248001
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Contact Us should mount 1`] = `
<div
  className="row pattern-black contact logo-container"
  id="contact"
>
  <div
    className="container"
  >
    <h2
      className="sectionTitle f-damion c-white mTop100"
    >
      Get in Touch!
    </h2>
    <h6
      className="f-robotoSlab fw-400 c-turquoise t-center mBottom30"
    >
      Be the first to know about our Events, Artist, Music
    </h6>
    <form>
      <div
        className="col-lg-8 col-centered"
      >
        <div
          className="col-lg-6"
        >
          <span
            className="input input--kozakura"
            style={
              Object {
                "height": "auto",
                "minHeight": 120,
              }
            }
          >
            <input
              className="input__field input__field--kozakura"
              id={undefined}
              maxLength={25000}
              name="name"
              onBlur={[Function]}
              onChange={[Function]}
              onFocus={[Function]}
              onKeyUp={undefined}
              pattern={undefined}
              placeholder=""
              required={true}
              tabIndex={undefined}
              type="text"
            />
            <label
              className="input__label input__label--kozakura"
              htmlFor="input-9"
            >
              <span
                className="input__label-content input__label-content--kozakura"
                data-content="Name"
              >
                Name
              </span>
            </label>
            <svg
              className="graphic graphic--kozakura"
              height="100%"
              preserveAspectRatio="none"
              viewBox="0 0 1200 60"
              width="300%"
            >
              <path
                d="M1200,9c0,0-305.005,0-401.001,0C733,9,675.327,4.969,598,4.969C514.994,4.969,449.336,9,400.333,9C299.666,9,0,9,0,9v43c0,0,299.666,0,400.333,0c49.002,0,114.66,3.484,197.667,3.484c77.327,0,135-3.484,200.999-3.484C894.995,52,1200,52,1200,52V9z"
              />
            </svg>
          </span>
        </div>
        <div
          className="col-lg-6"
        >
          <span
            className="input input--kozakura"
            style={
              Object {
                "height": "auto",
                "minHeight": 120,
              }
            }
          >
            <input
              className="input__field input__field--kozakura"
              id={undefined}
              maxLength={25000}
              name="email"
              onBlur={[Function]}
              onChange={[Function]}
              onFocus={[Function]}
              onKeyUp={undefined}
              pattern={undefined}
              placeholder=""
              required={true}
              tabIndex={undefined}
              type="email"
            />
            <label
              className="input__label input__label--kozakura"
              htmlFor="input-9"
            >
              <span
                className="input__label-content input__label-content--kozakura"
                data-content="Email"
              >
                Email
              </span>
            </label>
            <svg
              className="graphic graphic--kozakura"
              height="100%"
              preserveAspectRatio="none"
              viewBox="0 0 1200 60"
              width="300%"
            >
              <path
                d="M1200,9c0,0-305.005,0-401.001,0C733,9,675.327,4.969,598,4.969C514.994,4.969,449.336,9,400.333,9C299.666,9,0,9,0,9v43c0,0,299.666,0,400.333,0c49.002,0,114.66,3.484,197.667,3.484c77.327,0,135-3.484,200.999-3.484C894.995,52,1200,52,1200,52V9z"
              />
            </svg>
          </span>
        </div>
      </div>
      <div
        className="col-lg-10 col-centered"
      >
        <div
          className="col-lg-12"
        >
          <span
            className="input input--kozakura"
            style={
              Object {
                "height": "auto",
                "minHeight": 120,
              }
            }
          >
            <input
              className="input__field input__field--kozakura"
              id={undefined}
              maxLength={25000}
              name="message"
              onBlur={[Function]}
              onChange={[Function]}
              onFocus={[Function]}
              onKeyUp={undefined}
              pattern={undefined}
              placeholder=""
              required={true}
              tabIndex={undefined}
              type="text"
            />
            <label
              className="input__label input__label--kozakura"
              htmlFor="input-9"
            >
              <span
                className="input__label-content input__label-content--kozakura"
                data-content="Message"
              >
                Message
              </span>
            </label>
            <svg
              className="graphic graphic--kozakura"
              height="100%"
              preserveAspectRatio="none"
              viewBox="0 0 1200 60"
              width="300%"
            >
              <path
                d="M1200,9c0,0-305.005,0-401.001,0C733,9,675.327,4.969,598,4.969C514.994,4.969,449.336,9,400.333,9C299.666,9,0,9,0,9v43c0,0,299.666,0,400.333,0c49.002,0,114.66,3.484,197.667,3.484c77.327,0,135-3.484,200.999-3.484C894.995,52,1200,52,1200,52V9z"
              />
            </svg>
          </span>
        </div>
      </div>
      <div
        className="col-lg-12 t-center"
      >
        <a
          className=" gradientButton pink inverse mTop50"
          href={null}
          onClick={[Function]}
        >
          SEND MESSAGE
        </a>
      </div>
    </form>
  </div>
  <div
    className="container mTop50 mBottom50"
  >
    <div
      className="col-lg-8 col-sm-10 col-centered"
    >
      <div
        className="col-lg-12 col-sm-12"
      >
        <div
          className="row"
        >
          <div
            className="col-lg-4 col-sm-4 col-xs-12 t-center mBottom40"
          >
            <i
              className="flaticon-laptop15 c-pink iconSize45 mBottom20 hvr-buzz-out"
            />
            <h6
              className="f-raleway fw-700 c-white"
            >
              E-Mail
            </h6>
            <p
              className="c-lightGrey t-center"
            >
              <a
                className="textButtonWhite"
                href="mailto:info@thebigbandtheory.net"
              >
                info@thebigbandtheory.net
              </a>
            </p>
          </div>
          <div
            className="col-lg-4 col-sm-4 col-xs-12 t-center mBottom40"
          >
            <i
              className="flaticon-smart1 c-pink iconSize45 mBottom20 hvr-buzz-out"
            />
            <h6
              className="f-raleway fw-700 c-white"
            >
              Phone
            </h6>
            <p
              className="c-lightGrey t-center"
            >
              +91-8882003835
            </p>
          </div>
          <div
            className="col-lg-4 col-sm-4 col-xs-12 t-center mBottom40"
          >
            <i
              className="flaticon-open79 c-pink iconSize45 mBottom20 hvr-buzz-out"
            />
            <h6
              className="f-raleway fw-700 c-white"
            >
              Address
            </h6>
            <p
              className="c-lightGrey t-center"
            >
              190 model colony araghar, 
              <br />
              Dehradun, Uttrakhand 248001
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
